### DISCUSSION TITLE
My contributions aren't shown

### DISCUSSION THREAD

**Discussion by Alexponomarev7** [community/community#50714] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I had such contirbutions:

https://github.com/ytsaurus/ytsaurus/commits?author=Alexponomarev7
https://github.com/ceph/ceph/pull/44321
https://github.com/WayneD/rsync/pull/389 

They are not shown in my profile

**Comment by otter-computer** [comment]
It looks like some of these commits were made with an email address that isn't linked to your profile. I'd recommend you check the email address used to author these commits (you can do this in your browser by adding `.patch` to the end of a commit URL!) and adding any missing emails. You don't need to verify them, only add them to your account!

An example URL to one of your commits:
https://github.com/ceph/ceph/commit/22fefb2338cfc4fcb03ece3cbf77aa964a7f17f2.patch

**Reply by Alexponomarev7** [reply]
@otter-computer Thank you! I did it, but nothing happens 😞

**Reply by Alexponomarev7** [reply]
All emails are linked now, but none of these three repo is visible here 
<img width="370" alt="image" src="https://user-images.githubusercontent.com/8061177/226949902-4e9dda3b-b2a6-4b57-8f72-839d784d72a7.png">

**Reply by otter-computer** [reply]
Oh sorry, you're talking about the achievements, not the contribution graph! 

Achievements only list milestones that you meet rather than listing every pull request that helped you meet them. 

You've already hit the bronze milestone with your 16th pull request to `edgarsjmavonyan/frieview`. I believe the next milestone you'd hit is your 128th pull request. Once you meet that you'll see it show up underneath this achievement.
